### ISSUE TITLE
MOUSE_BTN3 not always correctly recognized

### ISSUE THREAD

**Issue by ms-89** [mpv-player/mpv#3772]
A few weeks ago I started noticing an issue with seeking backwards and forwards.

MOUSE_BTN3 is wheel up
MOUSE_BTN4 is wheel down

So I use this in my input.conf to define scrolling:
```
MOUSE_BTN3 seek -30
MOUSE_BTN4 seek 30
```
When scrolling down one step, mpv forwards 30 seconds. If I scroll down multiple steps per second, mpv jumps accordingly multiple steps forward.

Now when I scroll up one step, mpv also **forwards** 30 seconds. Only if I scroll **multiple steps per second up** mpv notices it as **backwards**.

I'm running macOS 10.12.1 with mpv git-932ecbf, using a wired USB mouse.

```
> mpv --input-test --force-window --idle

(scrolling down 1 step)
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11

(scrolling up 1 step)
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11

(scrolling down multiple steps)
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11

(scrolling up multiple steps)
[input] Key MOUSE_BTN4 is bound to:
[input] 1.  'seek -10' in <builtin>:1 (default)
[input] 2.  'seek 30' in /Users/michael/.mpv/input.conf:11
[input] Key MOUSE_BTN3 is bound to:
[input] 1.  'seek 10' in <builtin>:1 (default)
[input] 2.  'seek -30' in /Users/michael/.mpv/input.conf:10
[input] Key MOUSE_BTN3 is bound to:
[input] 1.  'seek 10' in <builtin>:1 (default)
[input] 2.  'seek -30' in /Users/michael/.mpv/input.conf:10
[input] Key MOUSE_BTN3 is bound to:
[input] 1.  'seek 10' in <builtin>:1 (default)
[input] 2.  'seek -30' in /Users/michael/.mpv/input.conf:10
```

**Comment by wiiaboo** [comment]
Looks more like a shitty mouse than mpv's problem.

**Comment by ms-89** [comment]
You comment made me want to write something like "And why does it work in Chrome, Terminal, Finder and all other Applications?" - then I realized it doesn't. Small scrolling steps are always ignored.

After 1 hour searching on the web I found out that's an macOS Sierra bug where scrolling acceleration for pointing devices is enabled, even if it's not a trackpad or magic mouse and there is no way to disabled it. I enabled the beta updates and on macOS 10.12.2 Beta it's working fine now.

Sorry for the confusion.
